### PR TITLE
[WIP] Show textarea for template params that are specified as multiline

### PR DESCRIPTION
**Pull request by jwforres**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1312444

### CODE REVIEW

**Comment by jwforres** [comment]
Pending testing with API change from @bparees 

cc @spadgett

**Comment by spadgett** [comment]
Clean/install/build assets? The bindata diff is large.

File: assets/app/views/_templateopt.html [hunk 1]
```diff
@@ -29,6 +30,21 @@
...
+          class="form-control"
+          rows="4"
+          placeholder="{{ parameter | parameterPlaceholder }}"
+          ng-model="parameter.value"
+          ng-required="parameter.required && !parameter.generate"
+          ng-blur="cleared = false"
+          ng-trim="false"
+          autocorrect="off"
+          autocapitalize="off"
+          spellcheck="false"
```
Comment (spadgett) [inline assets/app/views/_templateopt.html]: Can't decide if we should remove the autocorrect / autocapitalize / spellcheck off attributes. If it's a textarea, it's often regular text. If it's a cert, you'd paste it in anyway?

**Comment by spadgett** [comment]
Just the one question, otherwise LGTM

File: assets/app/views/_templateopt.html (hunk 1 above)
Comment (jwforres) [inline assets/app/views/_templateopt.html]: Yeah I'll remove them, it could theoretically be used for someone to put in a description or something

**Comment by jwforres** [comment]
ok updated, so just waiting to test it

**Comment by spadgett** [comment]
LGTM

**Comment by danmcp** [comment]
Needs to be moved to new ui repo if still valid.

**Pull request closed without merging**